addon/mixins/adapter-fetch.js

```diff
@@ -165,6 +165,20 @@ export function determineBodyPromise(response, requestData) {
 }
 
 export default Ember.Mixin.create({
+/**
+ * @param {String} url
+ * @param {String} type
+ * @param {Object} _options
+ * @returns {Object}
+ * @override
+ */
+  ajaxOptions(url, type, _options) {
```

Comment:
Alternatively this can be expressed as:

```js
ajaxOptions(url, type, options = {}) {
  options.url = url;
  options.type = type
  return mungOptionsForFetch(options, this);
}
```